tests/doc/parse.js

```diff
@@ -75,6 +75,16 @@ describe('tags', () => {
   })
 })
 
+describe('custom string on node', () => {
+  test('custom null', () => {
+    YAML.scalarOptions.null.nullStr = 'None'
+    const doc = YAML.parse('a: null')
+    const str = YAML.stringify(doc, {simpleKeys: true})
+    expect(str).toBe('a: None\n')
+    expect(YAML.parse(str)).toEqual({a: null})
+  })
+})
```

Comment:
I don't think we should include an example that uses invalid YAML. Use for example `~` instead of `None`.

Reply:
ok, add the testcase for `~` and `''`